src/lozad.js

```diff
@@ -55,6 +64,15 @@ export default function (selector = '.lozad', options = {}) {
         load(elements[i])
         markAsLoaded(elements[i])
       }
+    },
+    triggerLoad(selectorToTrigger) {
```

Comment:
triggerLoad should simply refer to the default defined load function

Reply:
Actually, I think that `triggerLoad` should load an specific image, not all images tracked by lozad. That's why I use matches.

>It would be useful to be able to preload certain images in advance when certain events are triggered (say, a menu is opened).

It was what I understood from #42. Maybe @akxcv could provide some clarification.

Reply:
Hmm, I'm not quite sure which is right. The most important thing is the reference to the `load` function, and it also yields less overhead for lozad. I think @ApoorvSaxena is right here.

Reply:
@thiamsantos please make the changes as suggested to merge this PR

Reply:
@ApoorvSaxena sure. I just doesn't have time now to do that. But soon in the next week I will do it.

Reply:
👍